packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts

```diff
@@ -684,8 +684,8 @@ export function _makeMatchingOrder({
     staticExtradata: '0x',
     staticTarget: NULL_ADDRESS,
     taker: order.maker,
-    takerProtocolFee: new BigNumber(order.makerProtocolFee),
-    takerRelayerFee: new BigNumber(order.makerRelayerFee),
+    takerProtocolFee: new BigNumber(order.takerProtocolFee),
+    takerRelayerFee: new BigNumber(order.takerRelayerFee),
```

Comment:
@jmichel-bc do you remember why we did this?

Reply:
Nope, I know that it did it but  honestly the change doesn't make any sense to me!